### DISCUSSION TITLE
Welcome to free365 Discussions!

### DISCUSSION THREAD

**Discussion by KusakabeShi** [KusakabeSi/free365#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by obujbalokakram** [comment]
https://azzzzzz.lx.workers.dev/
That site not show password
Please solve the issue

**Reply by KusakabeShi** [reply]
This is just a collection site, I can't do anything except remove it from my collection.
Thank your report.

**Comment by obujbalokakram** [comment]
Please give me a site which provide free azure

**Reply by KusakabeShi** [reply]
I listed all the collections that I found on the internet. There are one provide it before, but get banned now. Sorry I can't help. But if you found one, PRs welcome.

**Comment by obujbalokakram** [comment]
Please found  a site 
Which provide azure

**Reply by KusakabeShi** [reply]
How about find it by your self?

**Reply by obujbalokakram** [reply]
I don't understand you

**Comment by obujbalokakram** [comment]
I mean I need a site Where to get free azure

**Reply by KusakabeShi** [reply]
I can't help. I didn't find any site provide free azure 4 student lives now.
